src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentCost = 40 * days;
+  const minRentDiscount = 20;
+  const maxRentDiscount = 50;
+
+  if (days >= 7) {
+    return rentCost - maxRentDiscount;
+  }
+
+  if (days >= 3) {
```

Comment:
The same for days. Better to clarify that these are days for different discounts